com.ibm.streamsx.topology/opt/python/packages/streamsx/topology/context.py

```diff
@@ -656,13 +660,14 @@ class JobConfig(object):
         job_config.add(cfg)
         context.submit('ANALYTICS_SERVICE', topo, cfg)
     """
-    def __init__(self, job_name=None, job_group=None, preload=False, data_directory=None, tracing=None):
+    def __init__(self, job_name=None, job_group=None, preload=False, data_directory=None, submission_parameters=None, target_pe_count=None, tracing=None):
```

Comment:
Changing the order of parameters breaks backwards compatibility.

Target pe count was not added as a parameter as I didn't want too many parameters on the constructor, since they can easily be set as properties of the object.